Packs/BitSight/Integrations/BitSightEventCollector/BitSightEventCollector_test.py

```diff
@@ -0,0 +1,487 @@
+import pytest
+from freezegun import freeze_time
+from CommonServerPython import DemistoException, CommandResults
+
+from BitSightEventCollector import (
+    Client,
+    to_bitsight_date,
+    findings_to_events,
+    resolve_guid,
+    fetch_events,
+    bitsight_get_events_command,
+)
+
+
+@pytest.fixture
+def mock_client(mocker):
+    """Create a mocked BitSight client for testing."""
+    mocker.patch.object(Client, "_http_request")
+
+    client = Client(base_url="https://api.bitsighttech.com", verify=True, proxy=False, auth=("test_api_key", ""))
+
+    return client
+
+
+@pytest.fixture
+def sample_findings():
```

Comment:
It's better to work with `.json` files - though the code _is cleaner_ when using it as a fixture, the json responses make it easier for future debugging beyond the UT, as they save the full expected API responses, even with fields that seem irrelevant at the time.
In any case, no need to change here, but please keep it in mind for the next PRs.